packages/orchestrator-ui-components/src/components/WfoForms/formFields/deprecated/VlanField.tsx

```diff
@@ -227,10 +227,8 @@ function Vlan({
             : undefined;
 
     let message = '';
-    if (!imsIsLoading) {
-        if (missingInIms) {
-            message = t('widgets.vlan.missingInIms');
-        } else if (portIsTagged && nsiVlansOnly) {
+    if (!imsIsLoading && subscriptionId) {
+        if (portIsTagged && nsiVlansOnly) {
```

Comment:
Can I just ignore changes to these formfields since they are in deprecated? If not, why is the frontend aware of nsivlans???

Reply:
Yes just ignore them